test/e2e/dra/dra.go

```diff
@@ -921,74 +1009,6 @@ var _ = framework.SIGDescribe("node")(framework.WithLabel("DRA"), framework.With
 			}).WithTimeout(f.Timeouts.PodDelete).Should(gomega.HaveField("Status.Allocation", (*resourceapi.AllocationResult)(nil)))
 		})
 
-		f.It("must be possible for the driver to update the ResourceClaim.Status.Devices once allocated", f.WithFeatureGate(features.DRAResourceClaimDeviceStatus), func(ctx context.Context) {
```

Comment:
This shouldn't have been in `singleNodeTests` because then it runs with and without kubelet. It gets moved to a better location such that it only runs once overall. I can also merge this separately.